### PR TITLE
Health endpoint

### PR DESCRIPTION
**Pull request by DragosDumitrache**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -34,24 +34,33 @@ func main() {
...
 		_, err = w.Write(indexPage)
 		if err != nil {
 			log.Fatal(err)
 		} else {
-			log.Printf(fmt.Sprintf("GET - / 200 from %s", r.UserAgent()))
+			log.Printf(fmt.Sprintf("GET - / 200 UserAgent: %s", r.UserAgent()))
 		}
 	})
-	mux.HandleFunc("/swagger.json", func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc("/swagger.json/", func(w http.ResponseWriter, r *http.Request) {
```
Comment (stephenbinns) [inline main.go]: not sure we want a trailing slash on this
```suggestion
	mux.HandleFunc("/swagger.json", func(w http.ResponseWriter, r *http.Request) {
```

File: main.go [hunk 2]
```diff
@@ -34,24 +34,33 @@ func main() {
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/" {
```
Comment (stephenbinns) [inline main.go]: Why is this needed?

Comment (DragosDumitrache) [reply main.go]: Actually not, it just means `/foo` will show `/` and that's why we had to do `/swagger.json/`, but removed them, the health endpoint should be enough (tried a few of the suggestions from S/O)

**Review by stephenbinns — APPROVED** [review]
